docker/docker-compose-postgres-test.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.6"
+
+services:
+  gxa-postgres-test:
+    image: postgres:10-alpine
+    container_name: $POSTGRES_HOST
+    networks:
+      - gxa
+    restart: always
+    command: -c max_wal_size=2GB
```

Comment:
Do we need this? For fixtures  WAL shouldn’t be an issue since they are small transactions.